packages/react-components/token-analyzer-preview/library/analysis.json

```diff
@@ -0,0 +1,974 @@
+        "tokens": [
+          {
+            "property": "backgroundColor",
+            "token": "tokens.colorNeutralBackground1",
+            "path": ["backgroundColor"]
+          },
+          {
+            "property": "color",
+            "token": "tokens.colorNeutralForeground1",
+            "path": ["color"]
+          },
+          {
+            "property": "fontFamily",
+            "token": "tokens.fontFamilyBase",
+            "path": ["fontFamily"]
+          },
+          {
+            "property": "borderRadius",
+            "token": "tokens.borderRadiusMedium",
+            "path": ["borderRadius"]
+          },
+          {
+            "property": "fontSize",
+            "token": "tokens.fontSizeBase300",
+            "path": ["fontSize"]
+          },
+          {
+            "property": "fontWeight",
+            "token": "tokens.fontWeightSemibold",
+            "path": ["fontWeight"]
+          },
+          {
+            "property": "lineHeight",
+            "token": "tokens.lineHeightBase300",
+            "path": ["lineHeight"]
+          },
+          {
+            "property": "transitionDuration",
+            "token": "tokens.durationFaster",
+            "path": ["transitionDuration"]
+          },
+          {
+            "property": "transitionTimingFunction",
+            "token": "tokens.curveEasyEase",
+            "path": ["transitionTimingFunction"]
+          }
+        ],
+        "nested": {
+          "':hover'": {
+            "tokens": [
+              {
+                "property": "backgroundColor",
+                "token": "tokens.colorNeutralBackground1Hover",
+                "path": []
+              },
+              {
+                "property": "borderColor",
+                "token": "tokens.colorNeutralStroke1Hover",
+                "path": []
+              },
+              {
+                "property": "color",
+                "token": "tokens.colorNeutralForeground1Hover",
+                "path": []
+              }
+            ]
+          },
+          "':hover:active'": {
+            "tokens": [
+              {
+                "property": "backgroundColor",
+                "token": "tokens.colorNeutralBackground1Pressed",
+                "path": []
+              },
+              {
+                "property": "borderColor",
+                "token": "tokens.colorNeutralStroke1Pressed",
+                "path": []
+              },
+              {
+                "property": "color",
+                "token": "tokens.colorNeutralForeground1Pressed",
+                "path": []
+              }
+            ]
+          }
+        },
+        "assignedVariables": ["rootBaseClassName"],
+        "isClassFunction": true
+      },
+      "useIconBaseClassName": {
+        "tokens": [
+          {
+            "property": "[iconSpacingVar]",
+            "token": "tokens.spacingHorizontalSNudge",
+            "path": ["[iconSpacingVar]"]
```

Comment:
For these, I think we'd need to look up and resolve this name which in this case is `--fui-Button__icon--spacing` a variable.